src/features/query-bar/__tests__/ColumnButtonOverlay-test.js

```diff
@@ -24,18 +34,28 @@ describe('features/query-bar/components/ColumnButtonOverlay', () => {
                 updatedColumns: columnsWithNewOrder,
             },
             {
-                description: 'Should not update state due to no destinationIndex',
+                description:
+                    'Should not update state due to no destinationIndex',
                 sourceIndex: 0,
                 updatedColumns: columns,
             },
-        ].forEach(({ description, sourceIndex, destinationIndex, updatedColumns }) => {
-            test(`${description}`, () => {
-                const wrapper = getWrapper();
-                wrapper.instance().onDragEnd(sourceIndex, destinationIndex);
+        ].forEach(
```

Comment:
instead of doing a forEach here could you instead use a jest.each?

Reply:
Will tackle this in a separate PR